packages/affiliates/test/mocha/dappmining.js

```diff
@@ -262,3 +262,260 @@ describe("DappMining", function() {
     });
   });
 });
+describe("DappMining V2", function() {
```

Comment:
These new tests copy most from v1, but subtly changes some of the scenarios to test that the lookback is working correctly